frontend/src/components/Review/Review.tsx

```diff
@@ -55,50 +57,73 @@ const ReviewComponent = ({ review }: Props): ReactElement => {
+                  </Grid>
+                ) : (
+                  <Grid item>
+                    <Typography>This review was collected by the Ithaca Tenants Union</Typography>
+                  </Grid>
+                )}
                 <Grid item>
-                  <HeartRating value={overallRating} readOnly />
+                  <Typography className={dateText}>{formattedDate}</Typography>
                 </Grid>
+              </Grid>
+              {overallRating !== null && (
                 <Grid item>
-                  <IconButton
-                    className={clsx(expand, {
-                      [expandOpen]: expanded,
-                    })}
-                    onClick={handleExpandClick}
-                    aria-expanded={expanded}
-                    aria-label="show more"
-                    size="small"
-                  >
-                    <ExpandMoreIcon />
-                  </IconButton>
+                  <Collapse in={expanded} timeout="auto" unmountOnExit>
+                    <CardContent>
+                      <DetailedRatings ratings={detailedRatings} />
+                    </CardContent>
+                  </Collapse>
                 </Grid>
+              )}
+              <Grid item container alignContent="center">
+                <Typography>
+                  {expandedText ? reviewText : reviewText.substring(0, 500)}
+                  {!expandedText && reviewText.length > 500 && '...'}
+                  {reviewText.length > 500 ? (
+                    <Button className={button} onClick={() => setExpandedText(!expandedText)}>
+                      {expandedText ? 'Read Less' : 'Read More'}
+                    </Button>
+                  ) : null}
+                </Typography>
               </Grid>
-              <Grid item>
-                <Typography className={dateText}>{formattedDate}</Typography>
-              </Grid>
-            </Grid>
-            <Grid item>
-              <Collapse in={expanded} timeout="auto" unmountOnExit>
-                <CardContent>
-                  <DetailedRatings ratings={detailedRatings} />
-                </CardContent>
-              </Collapse>
-            </Grid>
-            <Grid item container alignContent="center">
-              <Typography>{reviewText}</Typography>
-            </Grid>
-            {photos.length > 0 && (
-              <Grid container alignItems="center" justify="center">
-                <Grid item xs={12} sm={6}>
-                  <CardMedia component="img" alt="Apt image" image={photos[0]} title="Apt image" />
+              {photos.length > 0 && (
+                <Grid container alignItems="center" justify="center">
+                  <Grid item xs={12} sm={6}>
+                    <CardMedia
+                      component="img"
+                      alt="Apt image"
+                      image={photos[0]}
+                      title="Apt image"
+                    />
+                  </Grid>
```

Comment:
Just one more nit here. Currently we are only displaying the first photo for a review, but in the design there is a carousel for multiple photos:
<img width="505" alt="Screen Shot 2021-05-08 at 10 29 22 PM" src="https://user-images.githubusercontent.com/26571581/117558609-e3c0d280-b04c-11eb-9dd2-db411876d4ca.png">
 I don't think we should change this for this PR, but maybe we can include this in a future PR. Great code quality though!